torchvision/prototype/transforms/functional/_color.py

```diff
@@ -469,7 +469,7 @@ def equalize(inpt: features.InputTypeJIT) -> features.InputTypeJIT:
 
 def invert_image_tensor(image: torch.Tensor) -> torch.Tensor:
     if image.is_floating_point():
-        return _FT._max_value(image.dtype) - image  # type: ignore[no-any-return]
+        return 1.0 - image  # type: ignore[no-any-return]
```

Comment:
This is correct but if for some reason on the future we decide to have different treatment around `max_value` for floats, it would have to be replaced. No action needed but I would be OK with the extra unnecessary method call as well.

Reply:
Apart from here, this is at least hardcoded for `convert_dtype` for the float to int 

https://github.com/pytorch/vision/blob/436ff9a52a0c901aa1b9bb51c0e580ae7f7cd4c0/torchvision/prototype/transforms/functional/_meta.py#L355-L359

and int to float cases

https://github.com/pytorch/vision/blob/436ff9a52a0c901aa1b9bb51c0e580ae7f7cd4c0/torchvision/prototype/transforms/functional/_meta.py#L363

If we don't hardcode `1.0` here, we should also do the same there.

Reply:
Agreed, good spot. It's up to you how you will handle it TBH. Another way would be to just add a comment and say we hardcode to avoid the call. No strong opinions on my side.